util/container/Dockerfile

```diff
@@ -64,10 +64,13 @@ COPY --from=verilator /tools/verilator/${VERILATOR_VERSION} verilator/${VERILATO
 # Install (and cleanup) required packages (from apt-requirements.txt)
 # The list of extra packages is leftover from before this Dockerfile used
 # apt-requirements.txt
+#
+# This also adds `locales` and `locales-all` so we can set the locale to utf-8
 COPY apt-requirements.txt /tmp/apt-requirements.txt
 RUN apt-get update && \
     sed 's/#.*//' /tmp/apt-requirements.txt \
         | xargs apt-get install -y && \
+    apt-get install -y locales locales-all && \
```

Comment:
nit: Can you just add these to the `apt-get install` line below? Just cleaner to maintain one install line.

Reply:
No, because I hope the second install line will be going away at some point - given those aren't official project requirements. When they delete that line, I don't want `locales` and `locales-all` not being installed, because we will still need them as docker-specific dependencies.

Reply:
Sounds good.